src/fu-util-common.c

```diff
@@ -248,7 +248,7 @@ fu_util_is_interesting_device (FwupdDevice *dev)
 }
 
 gchar *
-fu_util_get_user_cache_path (const gchar *fn)
+fu_util_get_user_cache_path (const gchar *fn, GError **error)
 {
 	const gchar *root = g_get_user_cache_dir ();
 	g_autofree gchar *basename = g_path_get_basename (fn);
```

Comment:
I think my error comes from here because fn is NULL

Reply:
In that case we probably shouldn't merge this PR, but fix the case that was causing `fn` to be `NULL`